### ISSUE TITLE
Missing :output filter crashes

### ISSUE THREAD

**Issue by apotonick** [trailblazer/trailblazer-activity-dsl-linear#6]
`NoMethodError: undefined method `call' for nil:NilClass`

**Comment by emaglio** [comment]
That could be the new error that I get in #5

**Comment by apotonick** [comment]
Hey cool, we don't need Slack anymore, we can just chat here! :sweat_smile:

**Comment by apotonick** [comment]
Quick fix: 
```
step :bla, input: ->(ctx, *) { ... }, 
output: ->(scope, ctx) { ctx[:a] = scope[:b]; ctx }
```

**Comment by apotonick** [comment]
Fixed in 0.1.4.
